model/version/version.go

```diff
@@ -75,3 +76,54 @@ var (
 	BuildStatusActivateAtKey = bsonutil.MustHaveTag(BuildStatus{}, "ActivateAt")
 	BuildStatusBuildIdKey    = bsonutil.MustHaveTag(BuildStatus{}, "BuildId")
 )
+
+type DuplicateVersionsID struct {
+	Hash      string `bson:"hash"`
+	ProjectID string `bson:"project_id"`
+}
+
+type DuplicateVersions struct {
+	ID       DuplicateVersionsID `bson:"_id"`
+	Versions []Version           `bson:"versions"`
+}
+
+func FindDuplicateVersions(since time.Time) ([]DuplicateVersions, error) {
+	pipeline := []bson.M{
+		bson.M{
```

Comment:
these types are redundant and not necessary, as it turns out.

Reply:
What are you thinking of using as a return type here? I could use a []Version, but I think it's useful to keep the duplicates with the same project/hash together.

Reply:
This is just a syntax point, you can omit the bson.M from the slice members because that type is implied by the type of the slice (this also applies to maps)

Reply:
Oh, weird. On the mobile view, it looked like this comment was attached to the DuplicateVersions type.